test_helpers/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+#Copyright (C) Microsoft Corporation. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set(c_util_test_helpers_h_files
+    inc/c_util_test_helpers/rc_string_test_type.h
+)
+
+set(c_util_test_helpers_c_files
+    src/rc_string_test_type.c
+)
+
+if(WIN32)
+set(c_util_test_helpers_h_files
+    ${c_util_test_helpers_h_files}
+    inc/c_util_test_helpers/uuid_test_type.h
+)
+
+set(c_util_test_helpers_c_files
+    ${c_util_test_helpers_c_files}
+    src/uuid_test_type.c
+)
+endif()
+
+include_directories(${CMAKE_CURRENT_LIST_DIR}/inc)
+
+add_library(c_util_test_helpers ${c_util_test_helpers_h_files} ${c_util_test_helpers_c_files})
+target_link_libraries(c_util_test_helpers rpcrt4 c_pal c_util_reals)
```

Comment:
Should `rpcrt4` be under a `if(WIN32)` check?

Reply:
It probably should not be here in the first place, removed.